src/vivarium_gates_nutrition_optimization/components/pregnancy.py

```diff
@@ -0,0 +1,21 @@
+from vivarium_public_health.disease import DiseaseModel, DiseaseState, SusceptibleState
+
+from vivarium_gates_nutrition_optimization.constants import data_keys, models
+
+
+def Pregnancy():
+    not_pregnant = SusceptibleState(models.PREGNANCY_SUSCEPTIBLE_STATE_NAME)
```

Comment:
It looks like this actually needs to be  SusceptibleState(models.PREGNANCY_STATE_NAME)

Reply:
fixed